composer.lock

```diff
@@ -11647,5 +11647,5 @@
     "platform-overrides": {
         "php": "7.4"
     },
-    "plugin-api-version": "2.2.0"
+    "plugin-api-version": "2.1.0"
```

Comment:
is this ok?

Reply:
is the result of updating version of `oat-sa/extension-tao-itemqti` in composer to `29.14.4.3` and running `composer update oat-sa/extension-tao-itemqti` 
Not sure why the version decreased to be honest...  ![image](https://user-images.githubusercontent.com/60346520/201946402-a19ee466-3277-49bf-987b-4bf493d81fb0.png)

Reply:
Maybe it is ok. Look at the previous PR https://github.com/oat-sa/tao-community/commit/7b64b2b658c18e8b808e14635b665f4f534db45a